src/context.c

```diff
@@ -69,6 +69,13 @@ xkb_context_include_path_append(struct xkb_context *ctx, const char *path)
     return 0;
 }
 
+const char *
+xkb_context_include_path_get_system_default(struct xkb_context *ctx)
```

Comment:
I think we should remove the "default" from the name - it is the system path itself, the default system path would be `DFLT_XKB_CONFIG_ROOT` I suppose.

Reply:
renamed to `xkb_context_include_path_get_system_path()` because I can't think of a better name. Maybe santa brings me an epiphany for christmas ;)